src/handle-data.ts

```diff
@@ -47,7 +47,7 @@ export function getTotalPosition(layout: TableLayout) {
  * Gets the totals text for a column
  */
 export function getTotalInfo(layout: TableLayout, colIdx: number, pageColIdx: number, numDims: number) {
-  if (colIdx >= numDims) return layout.qHyperCube.qGrandTotalRow[colIdx - numDims]?.qText;
+  if (colIdx >= numDims) return layout.qHyperCube.qGrandTotalRow[colIdx - numDims]?.qText || '';
```

Comment:
Consider using nullish coalescing operator `??` instead of `||` to deal with `null` and `undefined`. There can be some unexpected interactions with `||` when you have falsy values.

Not really expected anything to change here but in general I'd prefer ?? over || when you want to deal with null and undefined.

Reply:
you are right, this is a numerical value so it could be 0. good catch